hack/latest-binaries.sh

```diff
@@ -4,23 +4,75 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
+
+BUCKET="amazon-eks"
+REGION="us-west-2"
+PROFILE=""
```

Comment:
The region and profile would already be pulled in with the standard CLI environment variables (`AWS_PROFILE`, `AWS_REGION`).

Reply:
I can see the case for AWS_PROFILE. But `aws_region` is already is exposed as a `PKR_VAR_` and so by not explicitly using `--region <region>` in `latest-binaries` it creates confusing behavior

Reply:
You can call the script in the Makefile like:
```
AWS_REGION=$(aws_region) hack/latest-binaries.sh
```
I think this is just as clear as:
```
hack/latest-binaries.sh --region $(aws_region)
```
But with the former you don't need to add CLI parsing to this script.

Reply:
I agree those are equally readable. Now, as I go about making these changes. Do you have a preference on how to conditionally export `AWS_REGION` and `BINARY_BUCKET_NAME` in the Makefile?

One of the reasons I took this approach is that my Make isn't that strong, and this was the most readable to approach